lib/ldclient-rb/ldclient.rb

```diff
@@ -39,30 +39,35 @@ def initialize(api_key, config = Config.default)
       @worker = create_worker()
     end
 
+    def flush()
```

Comment:
I think this may cause thread-safety issues if there is a regularly-scheduled flush happening at the same time as a manual flush.